Jenkinsfile

```diff
@@ -293,19 +299,19 @@ timestamps {
 					// TODO parallelize the iOS/Android portions?
 					ansiColor('xterm') {
 						timeout(15) {
-							def buildCommand = "npm run clean -- --android-ndk ${env.ANDROID_NDK_R16B} --android-sdk ${env.ANDROID_SDK}"
+							def buildCommand = "npm run clean --"
 							if (isMainlineBranch) {
 								buildCommand += ' --all'
 							}
 							sh label: 'clean', script: buildCommand
 						} // timeout
 						timeout(15) {
-							def buildCommand = "npm run build -- --android-ndk ${env.ANDROID_NDK_R16B} --android-sdk ${env.ANDROID_SDK}"
+							def buildCommand = "npm run build --"
 							if (isMainlineBranch) {
 								buildCommand += ' --all'
 							}
 							sh label: 'build', script: buildCommand
-							recordIssues(tools: [clang(), java()])
+							recordIssues(tools: [checkstyle(), clang(), java()])
```

Comment:
So this assumes that build/clean will write xml files with the results somewhere. (The default is to assume finding them via the pattern `**/checkstyle-result.xml`) It's unclear if the gradle build does that already (you mentioned an html report)

Reply:
So I checked out this PR and ran it, it appears we'll need to tweak this to be:
```groovy
recordIssues(tools: [checkstyle(pattern: 'android/**/build/reports/checkstyle/checkJavaStyle.xml'), clang(), java()])
```

The issue is that checkstyle is also being run during the `npm test` call earlier as well (and I assume that's why you're also trying to just set the NDK/SDK values in environment variables). So I think it may make sense to recordIssues after the npm test call above and just try and record java/clang issues after build?